src/mudlet-lua/lua/Other.lua

```diff
@@ -1045,8 +1040,64 @@ function packageDrop(event, fileName, suffix)
     -- Cyan and Orange; see cTelnet::postMessage for color comparison
   end
 end
+
+
+--- Installs package from url
+-- @param url
+function installPackageFromUrl(url)
```

Comment:
I'd just make `installPackage()` handle it instead of a separate function, that would be more intuitive to work with. You can override the C++ one in Lua in case the `url` is an online one. Search for `local old` in Lua for examples of Lua functions improving on the core C++ ones.